data_models/data_refinery_models/models.py

```diff
@@ -119,7 +117,7 @@ class ProcessorJob(TimeTrackedModel):
 
     # This point of this field is to identify which worker ran the job.
     # A few fields may actually be required or something other than just an id.
-    worker_id = models.CharField(max_length=256)
+    worker_id = models.CharField(max_length=256, null=True)
```

Comment:
That would be bad. However I am going to force this to get set by requiring every job to call a provided `start_job` function which will set this.

The reason I don't want to enforce this at the database level though is because I want to be able to create the job record before the job is picked up. Until the job is picked up by a worker, we can't know what the `worker_id` is.